modules/log-upgrade/6x_deleting-the-clusterlogging-instance.adoc

```diff
@@ -0,0 +1,41 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-05-20
+:_mod-docs-content-type: PROCEDURE
+
+[id="deleting-the-clusterlogging-instance_{context}"]
+= Deleting the ClusterLogging instance
+
+Delete the ClusterLogging instance because it is no longer needed in Logging 6.x.
+
+.Prerequisites
+* You have administrator permissions.
+* You installed the {oc-first}.
+
+.Procedure
+* Delete the ClusterLogging instance.
++
+[source,terminal]
+----
+$ oc delete clusterlogging <CR name> -n <namespace>
+----
+
+.Verification
+
+. Verify that no collector pods are running by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -l component=collector -n <namespace>
+----
+
+. Verify that no Check that no `clusterLogForwarder.logging.openshift.io` custom resource (CR) exists by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -l component=collector -n <namespace>
```

Comment:
This command is checking again the previous step, if pods are running, but not if a clusterLogForwader CR exists. I'd suggest to use:
```
$ oc get clusterlogforwarders.logging.openshift.io -A
```

Reply:
done